### PR TITLE
Fix intermittent k8s Auth

### PR DESCRIPTION
**Pull request by virajrk**
Update golang - 1.19
Update Kubectl to 1.21
Add a hack to wait until kube context is set

**Is this a BUG REPORT or FEATURE REQUEST?**:
Bug Fix.
Ref - https://github.com/keikoproj/flippy/issues/12

### CODE REVIEW

**Comment by virajrk** [comment]
Ref - https://github.com/kubernetes/kubernetes/issues/65785#issuecomment-865173559

Testing - 
Verified unit test. 
Deployed PR image to test cluster. Validated functionality.

File: install/startup.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Wait for Kubernetes context to be set and ready"
+
+max_retry_count=5
+kube_system_pod_count=$(kubectl get pod -n kube-system | wc -l)
```
Comment (vinay-g) [inline install/startup.sh]: this will count the header as well like use `--no-headers` to list only the pods

File: install/startup.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Wait for Kubernetes context to be set and ready"
+
+max_retry_count=5
```
Comment (vinay-g) [inline install/startup.sh]: Should we make this configurable via ENV ? so we can control the delay without creating a new image

**Review by vinay-g — CHANGES_REQUESTED** [review]

**Review by vinay-g — APPROVED** [review]

**Review by vrushalijoshi — APPROVED** [review]
looks good

**Pull request merged**